builder-classic-22/builder.toml

```diff
@@ -0,0 +1,111 @@
+description = "Classic (shimmed) buildpack builder for Heroku-22 stack, based on ubuntu:22.04 base image"
+
+[stack]
+id = "heroku-22"
+build-image = "heroku/heroku:22-cnb-build"
+run-image = "heroku/heroku:22-cnb"
+
+[lifecycle]
+version = "0.14.0"
+
+[[buildpacks]]
+  id = "heroku/java"
```

Comment:
Should these shimmed CNBs have different IDs to their non-shimmed counterparts?

eg:
```toml
  id = "heroku/java-classic"
```

Reply:
I tried; it didn't work. The ids here need to match the id of the downloaded buildpack. If we changed the `id` here to `heroku/java-classic`, we'd need to change it in the shim to match. If this is something we want (which I'm not opposed to), I'd rather do it in a supplemental PR.

Reply:
> The ids here need to match the id of the downloaded buildpack.

Ah I didn't realise. In which case yeah agree we should defer (if we decide to even do it).